### DISCUSSION TITLE
How to user checkpoint in Flink-cdc-3.0.0

### DISCUSSION THREAD

**Discussion by spihiker** [apache/flink-cdc#2891] [Q&A]
Start a task as following:
    ./bin/flink-cdc.sh ./conf/mysql-2-doris.yaml

Question:
When start a task,How to specify checkpoint?
 
                                                         Thanks for you help.

**Comment by Sirivasv** [comment]
I am seeing that in the docs via the CDC connector in the env object you can enable checkpointing like this:
https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc.html#datastream-source
It is not stated but maybe in the pipeline you can set it up according to the options available for the yaml file:
https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc.html#datastream-source
Via the chunk options.

**Reply by spihiker** [reply]
Hi, Sirivasv:
     I added the following line of configuration in mysql-2-doris.yaml:

fromSavepoint: /tmp/savepoint/savepoint-9dfb3b-81a4e3071c3d.

But when I start a job using ./bin/flink-cdc.sh ./conf/mysql-2-doris.yaml.It is   effective。

                                                    Thanks for you help.

**Comment by ouliusheng263** [comment]
I have the smae question
Have you solved this problem?

**Reply by spihiker** [reply]
No。Wait for the next version.
